proto/api.proto

```diff
@@ -532,12 +532,12 @@ service Api {
 
   rpc GetPerpContracts(GetPerpContractsRequest) returns (GetPerpContractsResponse) {
     option (google.api.http) = {
-      get: "/api/v1/trade/perp/contracts"
+      get: "/api/v1/market/perp/contracts"
```

Comment:
can you also update the version at the top of this file? says 1.1 right now, should be 1.6